src/main/kotlin/app/extractors/PythonExtractor.kt

```diff
@@ -34,4 +46,19 @@ class PythonExtractor : ExtractorInterface {
 
         return libraries.toList()
     }
+
+    override fun getLineLibrary(line: String): String {
```

Comment:
This function not used and it doesn't account detected imports. Did it make intentionally?

Reply:
It's a basic piece that returns the result of a classifier which we then should use in commit stat aggregating by intersection with detected imports.

Reply:
I think that it's better to select library according to detected imports inside this function because in such way we don't lose library that is actually imported but has smaller score than a library which isn't imported.

Reply:
I understand it, but my current working approach is when the classifier returns bunch of libraries based on the output probabilities(if some libraries have almost the same probability as the library with the highest prob we return them too). Anyway I'll think of how to improve the quality by introducing imported libraries. Thanks!